auction9-backend/handler.js

```diff
@@ -250,3 +252,46 @@ export const realizeFinishedAuction = async (event, context) => {
     });
   }
 };
+
+/* postNewBid - creates new auction
+ * POST: /auctions/id/new
+ */
+export const postNewBid = async (event, context) => {
+  try {
+    let reqBody = JSON.parse(event.body);
+    let newBid = reqBody.newBid;
+    let currentAuctionPrice = await mysql.query('SELECT price FROM tbl_auction WHERE auctionID=?', [reqBody.auction.auctionID]);
```

Comment:
`auctionId` should be used from path parameters

Reply:
Okay changed